Bangazon/Models/ComputerEmployee.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
I'm not sure you're going to need this model, but I'm not going to ask you to remove it....you *might* need it :)

Reply:
ok